doc-content/jbpm-docs/src/main/asciidoc/DesignBuildCases/case-management-clean-case-log-con.adoc

```diff
@@ -1,7 +1,7 @@
 [id='case-management-clean-case-log-con']
 = Case log removal from the database
 
-Use the `CaseLogCleanupCommand` to clean up cases, such as canceled cases that are using up database space. The `CaseLogCleanupCommand` command contains logic to automatically clean-up all or selected cases.
+Use `CaseLogCleanupCommand` to clean up cases, such as canceled cases that are using up database space. The `CaseLogCleanupCommand` command contains logic to automatically clean-up all or selected cases.
```

Comment:
As above, I do not agree with removal of the article.

Reply:
See my previous comment.